packages/shared-models/src/ymodels.ts

```diff
@@ -1031,6 +1055,18 @@ export class YMarkdownCell
       attachments: this.getAttachments()
     };
   }
+
+  /**
+   * Whether or not the cell is rendered.
+   */
+  get rendered(): boolean | null {
+    return this.ystate.get('rendered');
+  }
+  set rendered(rendered: boolean | null) {
+    this.transact(() => {
```

Comment:
This shouldn't be tracked by the undomanager. So it would be best to do this without a transaction.

Reply:
Thanks, fixing it :)